src/init.cpp

```diff
@@ -736,8 +736,8 @@ static bool AppInitServers()
     StartRPC();
     if (!StartHTTPRPC())
         return false;
-    if (gArgs.GetBoolArg("-rest", DEFAULT_REST_ENABLE) && !StartREST())
-        return false;
+    if (gArgs.GetBoolArg("-rest", DEFAULT_REST_ENABLE))
```

Comment:
Nit: if you're modifying this code anyway, update it to follow the style guide and put braces around then clause which isn't on the same line.

Reply:
Done